app/webpacker/lib/utils/competition-table.js

```diff
@@ -94,7 +94,7 @@ export function numberOfDaysAfter(competition, refDate) {
 
   const numberOfDays = parsedStartDate.diff(parsedRefDate, 'days').days;
 
-  return Math.ceil(Math.abs(numberOfDays));
+  return Math.floor(Math.abs(numberOfDays));
```

Comment:
Could you add a comment explaing why `floor` makes sense here?
Otherwise in the future, somebody might find an unrelated bug and just change this back to `ceil` accidentally.